.github/workflows/ci.yml

```diff
@@ -6,17 +6,26 @@ jobs:
 # whenever you do end up solving this issue so that I know how many years have passed.
 # Until that time, please manually run the test suite before pushing. Thanks. 
 #   - ValentijnvdBeek
-#  test:
-#    runs-on: ubuntu-latest
-#    steps:
-#      - name: Install Go
-#        uses: actions/setup-go@v2
-#        with:
-#          go-version: 1.16
-#      - name: Checkout code
-#        uses: actions/checkout@v2
-#      - name: Build
-#        run: go test ./...
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Check version
+        run: |
```

Comment:
I understand that this is here for debugging reasons. However would it not be better to remove this step before merging it? Or is there something I am missing?

Reply:
@ValentijnvdBeek 
Ah, It's my fault. That is just using for debugging, Then, I removed it.